developer-docs-site/docs/nodes/ait/ait3-leaderboard-metrics.md

```diff
@@ -0,0 +1,45 @@
+---
+title: "AIT-3 Leaderboard Metrics"
+slug: "ait-3-leaderboard-metrics"
+---
+
+# AIT-3 Leaderboard Metrics
+
+This document explains how the rewards for AIT-3 validator are evaluated and displayed on the [AIT-3 Validator Status](https://community.staging.gcp.aptosdev.com/leaderboard/it3?sort=-rewards_growth) page. 
+
+## How are rewards calculated
+
+:::tip Staking documentation
+For a backgrounder on staking with explanations of epoch, rewards and how to join and leave validator set, see the [Staking](/concepts/staking.md). 
+:::
+
+- For the AIT-3 the epoch duration is 7200 seconds, i.e., two hours.
+- An epoch starts with a finalized validator set. During the epoch, only validators in this validator set will vote. 
+- During the epoch, following the process described in [Validation on the Aptos blockchain](/concepts/staking#validation-on-the-aptos-blockchain), a validator is selected as a leader to make a proposal. Because the validator set is unchanged during the course of an epoch, this will result in a validator being selected multiple times as a leader in an epoch.
+-  On successful proposals, i.e., proposals achieving the quorum consensus, the leaders earn rewards based on their stake and on the reward rate that is configured on-chain. The reward rate is the same for every validator.
+-  If all the proposals in an epoch achieve quorum consensus, a validator earns the maximum reward for the epoch. **Rewards are given only to the leader validators, and not to the voters.**
+-  On failed proposals, i.e., a proposal that did not achieve the quorum consensus, the leaders do not earn any reward for that proposal.
+-  If all the proposals in an epoch fail, a validator earns zero rewards for that epoch.
```

Comment:
rewarding only happens at the end of the epoch, not on every block.
We multiple maximal possible reward with your proposal success rate. 
So someone with 8 successful and 2 failed proposals will get 80% of max reward.

But another validator with 80 successful and 20 failed, will also get 80% of max reward.


or two nodes with no failures, will get the same % of the max reward - for example one with 10 and another with 100 successful


max_reward does depend on the staking power itself, the more you stake, the more you can earn in absolute terms (because you can earn the same interest rate)